scripts/main.coffee

```diff
@@ -19,11 +19,11 @@ module.exports = (robot) ->
   # )
 
   robot.hear /.*/i, (res) ->
-      message = res.message.text.match(/\+[^*\s]+/)
+      message = messages.getAction(res.message.text)
       for key, val of config.monitor
           if(message != null)
-              if(key == message[0])
-                  for de in config.monitor[message[0]].dest
+              if(key == message)
+                  for de in config.monitor[message].dest
```

Comment:
I would even pass the config to the function `getLogs(text, config)` to avoid iterating here. Eg:
```
let getLogs(text, config) => {
  let monitor = Object.values(config.monitor)
  let act
  monitor.forEach(action) =>{
    act = text.find(action)
  }
  if (!act) {
    return
  }
  ....
  do the other stuff 
}
```